docs/content/en/install/eks/_index.md

```diff
@@ -0,0 +1,195 @@
+
+Apache Cassandra can be deployed across a variety of [hardware profiles](https://cassandra.apache.org/doc/latest/cassandra/operating/hardware.html). Consider the hardware requirements of Apache Cassandra when defining EKS node groups. Determine if you are planning on running a single Cassandra instance per node or multiple instances per node. If you are planning on running multiple instances per node, you will need to ensure that the node has enough resources to support the number of instances you are planning on running. For example, if you are planning on running 3 Cassandra instances per node, you will need to ensure that the node has enough CPU, memory, and storage to support 3 Cassandra instances.
+
+## Dependencies
+
+1. [Cert Manager](https://cert-manager.io/docs/installation/kubernetes/) - Cert Manager provides a common API for management of TLS certificates. It provides interfaces for communicating with a variety of Certificate Authority backends, including self-signed, Venafi, and Vault. K8ssandra Operator uses this API for certificates used by the various K8ssandra components.
+
+## Installation Type
+
+K8ssandra Operator may be deployed in one of two modes. `Control Plane` mode is the default method of installation. In this mode K8ssandra Operator watches for the creation and changes to `K8ssandraCluster` custom resources. When this mode is active it may create Cassandra resources within the local Kubernetes cluster **and / or** remote Kubernetes clusters (in the case of multi-region) deployments. When using K8ssandra Operator you should only have one instance running in `Control Plane` mode. Kubernetes clusters acting as remote regions for Cassandra deployments should be run in `Data Plane` mode. In this mode K8ssandra Operator will not directly reconcile `K8ssandraCluster` resources.
+
+For example consider the following scenarios:
+
+* Single region, shared-cluster - K8ssandra Operator is deployed in `Control Plane` mode on the cluster. It can manage the creation of `K8ssandraCluster` objects and reconcile them locally into Cassandra clusters and nodes.
+* Single region, dedicated cluster roles - K8ssandra Operator is deployed in `Control Plane` mode on a small, dedicated, k8s cluster. A separate cluster running in `Data Plane` mode contains larger instances which will be used to run Cassandra. This creates a separation between roles for each cluster (management vs production).
+* Multiple region, shared cluster - One region is configured with K8ssandra Operator running in `Control Plane` mode, this cluster will run both the management operators and Cassandra nodes. Additional regions are configured in `Data Plane` mode focusing solely on Cassandra nodes and services.
+* Multiple region, dedicated clusters - This model is similar to the last with the exception of a dedicated management cluster running in one of the regions. This follows the same pattern as the single region, dedicated cluster roles scenario.
```

Comment:
```suggestion
* Single-region, shared-cluster - K8ssandra Operator is deployed in `Control-Plane` mode on the cluster. It can manage the creation of `K8ssandraCluster` objects and reconcile them locally into Cassandra clusters and nodes.
* Single-region, dedicated cluster roles - K8ssandra Operator is deployed in `Control-Plane` mode on a small, dedicated, K8s cluster. A separate cluster running in `Data-Plane` mode contains larger instances in which to run Cassandra. This creates a separation between management vs. production roles for each cluster.
* Multiple-region, shared cluster - One region is configured with K8ssandra Operator running in `Control-Plane` mode. This cluster runs both the management operators and Cassandra nodes. Configure additional regions in `Data-Plane` mode to focus solely on Cassandra nodes and services.
* Multiple-region, dedicated clusters - This scenario is similar to a shared cluster with the exception of a dedicated management cluster running in one of the regions. This follows the same pattern as the single-region, dedicated cluster roles scenario.
"This follows the same pattern as the single-region, dedicated cluster roles scenario." this sentence seems superfluous and may be able to be deleted?

Reply:
Line 41  "This follows the same pattern as the single-region, dedicated cluster roles scenario." this sentence seems superfluous and may be able to be deleted?

Reply:
> this sentence seems superfluous and may be able to be deleted?

That's probably fair, I was drawing a parallel, but it isn't really necessary.

Adding dashes between `Control` / `Data` and `Plane` differs from the introduction of the modes previously in the doc. We should be uniform in our presentation. Which one is appropriate so I don't make this mistake again 😁 ?

Reply:
The dashes are uniformly used in the MC customer-facing doc and are used because they modify the noun 'plane'.

Reply:
Awesome, I wasn't indicating the suggestion was wrong. Rather that we must make sure it's applied uniformly across this page (and other pages) in the K8ssandra Operator docs. Would you like me to create a ticket indicating this is the standard to be used across our docs?

Reply:
Sure! consistency would be good!